Cargo.toml

```diff
@@ -40,6 +40,11 @@ walker = "^1.0.0"
 [target.x86_64-unknown-linux-gnu.dependencies.inotify]
 version = "^0.1"
 
+[target.x86_64-unknown-linux-gnu.dependencies.mio]
+git = "https://github.com/carllerche/mio.git"
```

Comment:
Sure. If there's an issue tracking this `mio` feature, could you link it here so we can keep track of when it makes it in a release?